weblate/static/vendor/daterangepicker.css

```diff
@@ -0,0 +1,417 @@
+/*
+  Modified daterangepicker.css
+  This is to align with the weblate design and colors
+ */

```

Comment:
As this is not a verbatim vendored file, please place it outside `vendor` dir. Moreover, did you check how it looks it both dark and light theme?

Reply:
I transferred the CSS file outside the vendor directory. I checked it and it still looks dark on the light theme. I improved it and now it supports light and dark themes.